metron-platform/metron-elasticsearch/README.md

```diff
@@ -81,3 +81,13 @@ curl -XPUT "http://${ELASTICSEARCH}:9200/${SENSOR}_index*/_mapping/${SENSOR}_doc
 '
 rm ${SENSOR}.template
 ```
+
+## Installing Elasticsearch Templates
+
+The stock set of Elasticsearch templates for bro, snort, yaf, error index and meta index are installed automatically during the first time install and startup of Metron Indexing service.
+
```

Comment:
Instead of saying 'in the rare case', it might be better to say:

"It is possible that Elasticsearch is not available when the Metron Indexing Service starts the Elasticsearch topology, in that case the Elasticsearch templates will not be installed."

Also, this doesn't document the scenarios you have been describing fully.

Reply:
Reworded per your suggestion. 

I believe the README section covers the current behavior with my change. If you are talking about the scenarios noted in the PR description, then these are captured in the existing ES README under the section [Using Metron with Elasticsearch 2.x](https://github.com/apache/metron/tree/master/metron-platform/metron-elasticsearch#using-metron-with-elasticsearch-2x). Please let me know which specific scenario you are referring to and I would be happy to include them.

Reply:
https://github.com/apache/metron/pull/817#issuecomment-340269215
https://github.com/apache/metron/pull/817#pullrequestreview-72703358

My impression from these comments is that automatic installation could still possibly happen.  That is what I'm referring to as being missing.  I may be confused though.

Reply:
Ah okay, this comment and review is the same scenario described in the README as:

> It is possible that Elasticsearch service is not available when the Metron Indexing Service startup, in that case the Elasticsearch template will not be installed. For such a scenario, an Admin can install the template manually from the Ambari UI by following the below flow:
> 
> Ambari UI -> Services -> Metron -> Service Actions -> Elasticsearch Template Install

Reply:
So I am def. confused.  I don't see how "It will try again next time X happens" as the same as Go and do it manually in ambari

Reply:
Okay, let me try to explain more using examples:

**Scenario 1 - Happy Path**
* Fresh install
* ES service up and running
* When the Indexing service comes up, it also installs the ES templates
* Admin can start ingesting into sensors, launch alerts UI and everything works

**Scenario 2 - ES service down**
* Fresh install
* For some reason, the ES service is down when the Indexing service is coming up
* We log a warning message in the Ambari install logs, and the Indexing service starts up fine.
* Once the ES service issue is resolved, the Admin needs to install the ES templates manually before s/he can start ingesting into sensors. This can be done in two ways:
1) Using the Ambari UI -> Services -> Metron -> Service Actions -> Elasticsearch Template Install
2) By stopping the Metron Indexing service from Ambari UI, and starting it again so that it can trigger the piece of code to install the template.

Now, from a documentation perspective, point 2 above is counter intuitive IMHO, since it would not make sense to ask the Admin to stop/start Indexing service in order to have the ES template installed. I have hence documented only the first option--which is also the same way it is done presently.

Let me know if this helps clarify.

Reply:
But, if that is the way it works, and is what will happen, you should documented it.  If it doesn't make sense enough to document, then change the behavior.

We have people all over the user's list with issues understanding what is going on with things.  

"Oh, the doc doesn't say it but we *do* try to install automatically next time or if you .... " isn't right.

Reply:
Fair enough, added both methods in the latest doc change.